tests/fast-integration/utils/index.js

```diff
@@ -1451,6 +1451,106 @@ function deleteEventingBackendK8sSecret(name, namespace='default') {
   return k8sDelete([secretJson], namespace);
 }
 
+/**
+ * Creates apirule for the service specified
+ * @param {string} name - name of the configmap
+ * @param {string} namespace - namespace where to create the configmap
+ * @param {string} svcName - service to expose as apirule
+ * @param {int} port - port of the service to expose as apirule
+ */
+async function createAPIRuleForService(name, namespace='default', svcName, port) {
```

Comment:
can you please rename to `...Api...` to stay in style with the existing function naming conventions?